src/InvertMouse/MainForm.Designer.cs

```diff
@@ -51,6 +52,8 @@ private void InitializeComponent()
             this.startStopByKeyCB = new System.Windows.Forms.CheckBox();
             this.startStopKeyTB = new System.Windows.Forms.TextBox();
             this.shieldIconPB = new System.Windows.Forms.PictureBox();
+            this.activeTitlePrefix = new System.Windows.Forms.TextBox();
```

Comment:
rename `activeTitlePrefix` to `activeTitlePrefixTB` for more consistency with other names